src/app/App.js

```diff
@@ -3,21 +3,63 @@ import Logo from './components/Logo';
 import createButtonRed from './components/ButtonRed';
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
 import createGameModeName from './components/GameModeName';
+import elemFactory from './utils/elementFactory';
+import render from './utils/render';
```

Comment:
Consider name changing to something more descriptive.

Reply:
Make a suggestion please.

Reply:
I actually have a problem with that, because when I started thinking about an alternative name I only came up with appendChildrenToParent(Element), but then I couldn't figure out the difference to parent.append(children). So I wonder if we need it.

Reply:
I was inspired by the react's render

Reply:
I understand. Interesting word. Well, consider it, but mostly if we really need it.